torch/_torch_docs.py

```diff
@@ -149,7 +149,7 @@ def merge_dicts(*dicts):
 absolute(input, *, out=None) -> Tensor
 
 Alias for :func:`torch.abs`
-""".format(**common_args))
+""")
```

Comment:
Just curious, is it all right to have it in this PR? What does it do?

Reply:
I thought having a separate PR just for cleaning this up is a waste of CI resources. Noticed the issue recently and added the cleanup here.

`common_args` is defined at the [top](https://github.com/pytorch/pytorch/blob/dcb5eb8d9b0daa277a083a4e5c5f7633a8fed862/torch/_torch_docs.py#L31) of the file to elaborate and describe a few of the common documentation things. But this is actually not needed in some of the cases where there is no curly braces, i.e. nothing to be formatted. (I guess someone copied it and the pattern was followed by others, that's how these were added initially to the docs for no reason)